config/application.rb

```diff
@@ -1,13 +1,6 @@
-require File.expand_path('../boot', __FILE__)
-
-require 'rails'
-# Pick the frameworks you want:
-require 'active_model/railtie'
-require 'active_job/railtie'
-require 'active_record/railtie'
-require 'action_controller/railtie'
-require 'action_mailer/railtie'
-require 'action_view/railtie'
+require_relative 'boot'
+
+require 'rails/all'
```

Comment:
Should we go back to the explicit enumeration of requires and leave off action_mailer + active_job + action_cable?

Reply:
I didn't think that through. I'll revert this and clean it up.